docs/build/configure/_index.md

```diff
@@ -15,10 +15,7 @@ Before you can program a smart machine, you must configure it.
 
```

Comment:
Not sure what happened but I think you restored the old/pre-R2D2 version of this file? This is also what's breaking the link/html tests. I would suggest copy pasting the version on R2D2 pr in and then making resource name changes from there!